### DISCUSSION TITLE
:nosearch: equivalent in markdown?

### DISCUSSION THREAD

**Discussion by lola-masson** [executablebooks/MyST-Parser#939] [Q&A] [answered]
Hi all,

I am trying to exclude a page from the search while still having it accessible in my documentation. This page is not included in any toctree but I want to access it with the precise URL when needed.

I saw that Sphinx with rst supports a [:nosearch:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#special-metadata-fields) field. Is there a way to do the same thing in markdown?

Thanks!

**Comment by chrisjsewell** [comment, marked as answer]
Heya, see https://myst-parser.readthedocs.io/en/latest/faq/index.html#sphinx-specific-page-front-matter

**Reply by lola-masson** [reply]
Excellent, thank you. For future reference, I added:
```
---
nosearch: true
---
```
to the top of the files

**Reply by chrisjsewell** [reply]
just to note also, the other way to do this, that should work, is to enable https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#field-lists,
then you would do exactly as you do in restructuredtext; adding `:nosearch:` at the top of the file

**Reply by lola-masson** [reply]
Ah, thank you! I was missing the `myst_enable_extensions = ["fieldlist"]`, this explains why it didn't work.
